src/alert.cpp

```diff
@@ -829,14 +829,16 @@ namespace libtorrent {
 	listen_failed_alert::listen_failed_alert(
 		aux::stack_allocator& alloc
 		, std::string const& iface
-		, tcp::endpoint const& ep
+		, libtorrent::address const& ip
```

Comment:
Yes, at least in my environment with cmake, clang seems to get confused with the types